daliuge-engine/dlg/manager/proc_daemon.py

```diff
@@ -242,18 +242,18 @@ def startMM(self):
         if self._zeroconf:
             nm_assigner = NMAssigner()
 
-            def nm_callback(zeroconf, service_type, name, state_change):
+            def _callback(zeroconf, service_type, name, state_change, adder, remover, accessor):
```

Comment:
I think `service_type` here should have "Node Manger" or "DIM" in it, so that you can use it in the logging statements in the callback, otherwise they'll always say "Node Manager", which could be confusing